src/ctap/credential_id.rs

```diff
@@ -47,6 +47,7 @@ struct CredentialSource {
     private_key: PrivateKey,
     rp_id_hash: [u8; 32],
     cred_protect_policy: Option<CredentialProtectionPolicy>,
+    cred_blob: Option<Vec<u8>>,
 }
 
 // The data fields contained in the credential ID are serizlied using CBOR maps.
```

Comment:
Just noticed I made a type here. Fix this for me please? :)
"serizlied"

Reply:
done